<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCutIntoPieces</class>
 <widget class="QDialog" name="DialogCutIntoPieces">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Разрезать изображение на N*M частей</string>
  </property>
  <widget class="QLabel" name="label_N">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>23</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>По горизонтали на :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_M">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>93</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>По вертикали на :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_N">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>23</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_M">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>93</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>169</width>
     <height>30</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>частей</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>частей</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_N_info">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>351</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_M_info">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>361</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
